index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const server = express();
+const cors = require('cors');
+const helmet = require('helmet');
```

Comment:
Not a requirement, but I'd like to see Morgan implemented just to give us some descriptive activity in the terminal.